src/interface/IAggregationRouter.sol

```diff
@@ -0,0 +1,47 @@
+/// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+struct SwapDescription {
+    IERC20 srcToken;
+    IERC20 dstToken;
+    address payable srcReceiver;
+    address payable dstReceiver;
+    uint256 amount;
+    uint256 minReturnAmount;
+    uint256 flags;
+    bytes permit;
+}
+
+/// @title Interface for making arbitrary calls during swap
+interface IAggregationExecutor {
+    /// @notice Make calls on `msgSender` with specified data
+    function callBytes(address msgSender, bytes calldata data) external payable; // 0x2636f7f8
```

Comment:
dangerous function remove the file if not used

Reply:
we need this